nix/overlays.nix

```diff
@@ -28,13 +28,12 @@ self: final: prev: {
             # for qtile migrate, not in nixpkgs yet
             propagatedBuildInputs = (old.propagatedBuildInputs or [ ]) ++ [
               pprev.libcst
-              pprev.dbus-fast
             ];
-            dependencies = prev.lib.filter (x: x != pprev.dbus-next) (old.propagatedBuildInputs or [ ]);
           }
         )).override
           {
             wlroots = prev.wlroots_0_17;
+            dbus-next = prev.python3Packages.dbus-fast;
```

Comment:
I'm not good on nix flakes but assigning the `dbus-fast` package to a variable called `dbus-next` doesn't seem like a great idea.

Reply:
I can't find any other way
If anyone knows I am happy to accept

Reply:
Can't we not do it like you did previously with `old.dependencies`?

Reply:
> Can't we do it like you did previously but with `old.dependencies`?

That doesn't work, we don't have influence on `old.dependencies`, tried dozens of times

Reply:
Maybe better to upstream the dbus-fast changes and update the flake?

Reply:
Maybe just copy over the whole dependencies list? It's only up until we make a new release anyways

Reply:
> Maybe just copy over the whole dependencies list? It's only up until we make a new release anyways

But what's the point if it can be done this way?

Reply:
Because overriding dbus-next is hacky

Reply:
Well it's for you to decide

Reply:
But keep in mind, doesn't matter if you set dependencies in overlay, since it gets ignored for some reason and still uses default package dependencies

Reply:
I tried setting dependencies list to empty list, but it still built with all the dependencies listed in Qtile package

Reply:
So far this is only way I actually got rid of dbus-next, if you have ideas you can try on my fork, since you are added as collaborator, I'll check changes in the morning